static/images/logos/logo-BP.svg

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please `git rm` all the old PNGs as well. Otherwise, we're cluttering up our repo with unused files.